packages/destination-actions/src/destinations/intercom/groupIdentifyContact/index.ts

```diff
@@ -0,0 +1,151 @@
+import { ActionDefinition, ModifiedResponse, RequestClient } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import { getUniqueIntercomContact } from '../util'
+import type { Payload } from './generated-types'
+
+interface IntercomCreateCompanyData {
+  id: string
+}
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Identify Company',
+  description: 'Create or update a company in Intercom and attach a contact.',
+  defaultSubscription: 'type = "group"',
+  fields: {
+    remote_created_at: {
+      type: 'datetime',
+      description: 'The time the company was created by you.',
+      label: 'Company Creation Time',
+      default: {
+        '@path': '$.traits.createdAt'
+      }
+    },
+    external_id: {
+      type: 'string',
+      description:
+        'A unique identifier for the contact generated outside Intercom. External ID is required if the role is `user` and email is blank.',
+      label: 'External ID',
+      default: {
+        '@path': '$.userId'
+      }
+    },
+    email: {
+      type: 'string',
+      description: "The contact's email address. Email is required if the role is `user` and external ID is blank.",
```

Comment:
@NolanChan is this still true? Or should it be?

```suggestion
      description: "The contact's email address. Email is required to attach a contact to a company if no External ID or Contact ID is provided.",
```